src/utils/keycloak.ts

```diff
@@ -12,6 +12,9 @@ import {
   formatQuerystringParameters,
 } from './url';
 
+const entriesPolyFill = (obj: any) =>
+  Object.keys(obj).map((key) => [key, obj[key]]);
```

Comment:
`fromEntries` should perform the reverse of this.

So given an `[string, unknown][]` (an `array` of entry) should return a `Record<string, unknown>` (an object with the former entry value as `key` and the latter entry value as `value`).

Something like this
```ts
const fromEntries = (entries: [string, unknown][]): Record<string, unknown> => entries.reduce((obj, [key, value]) => ({ 
...obj, 
[key]: value,
}), {});
```